.tekton/release-service-pull-request.yaml

```diff
@@ -163,18 +163,47 @@ spec:
       workspaces:
       - name: basic-auth
         workspace: git-auth
+    - name: sealights-go-instrumentation
+      runAfter:
+        - clone-repository
+      taskRef:
+        params:
+        - name: name
+          value: sealights-go-oci-ta
+        - name: bundle
+          value: quay.io/konflux-ci/tekton-catalog/task-sealights-go-oci-ta:0.1@sha256:5c31f32ea625b3a7419d7a6994bfa2cce477d3050c0c145dc14a47302feb253e
+        - name: kind
+          value: task
+        resolver: bundles
+      params:
+        - name: SOURCE_ARTIFACT
+          value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+        - name: component
+          value: '{{ repo_name }}'
+        - name: branch
+          value: '{{ source_branch }}'
+        - name: revision
+          value: '{{ revision }}'
+        - name: repository-url
+          value: '{{ repo_url }}'
+        - name: pull-request-number 
+          value: '{{pull_request_number}}'
+        - name: target-branch
+          value: '{{ target_branch }}'
+        - name: oci-storage
+          value: $(params.output-image).sealights.git
```

Comment:
Hi @jinqi7 could you please update the image sha and add `disable-token-save` parameter here similarly like [here](https://github.com/konflux-ci/image-controller/pull/188/files#diff-d05874b2e54b1e8bdd95161807da489eaaad9102d9c38927f0f0fff65ab679fdR178-R200) in on-pr pipeline and in push pipelinerun manifest as well?

Reply:
Updated, thanks